tests/pebble.rs

```diff
@@ -185,6 +150,82 @@ impl PebbleEnvironment {
         Ok(())
     }
 
+    /// Create and finalize an ACME order for the given `Account` and `Identifier`s using
+    /// the specified `ChallengeType`.
+    ///
+    /// The `provision_fn` will be invoked to set up the appropriate challenge response
+    /// based on the `Challenge`/`KeyAuthorization`.
+    ///
+    /// Returns the issued certificate chain unless an error occurs.
+    async fn complete_order<'a>(
```

Comment:
I feel like we should probably bite the bullet and figure out how to put minimally opinionated takes on this stuff in the library itself -- not much point in every downstream copy/pasting...

Reply:
I agree, but unless you had a stronger preference I think I'd prefer to land this test code and then take that on as a separate follow-up. WDYT?